src/library/simplified/extension/page-name-javascript-functions.js

```diff
@@ -124,7 +124,7 @@ window.utag.isFPymt = function(){
 window.utag.isFCO = function(){
     if(pageName.indexOf("page.Flight.Checkout.Confirmation") > -1  || pageName.indexOf("page.Flight.Checkout.Confirmation.Mobile,F,70") > -1 ||
         (b.lobDomain == 'AIR' && b.utagPageName == "page.Checkout.Confirmation") ||
```

Comment:
aren't lines 126 and the first half of the OR block of 127 redundant?
```(b.lobDomain == 'AIR' && b.utagPageName == "page.Checkout.Confirmation")``` should cover ```(b.lobDomain == 'AIR' && b.device_type == "Mobile" && (b.utagPageName == "page.Checkout.Confirmation"```

So all you should really need is 
```
if(pageName.indexOf("page.Flight.Checkout.Confirmation") > -1  || pageName.indexOf("page.Flight.Checkout.Confirmation.Mobile,F,70") > -1 ||
          (b.lobDomain == 'AIR' && (b.utagPageName == "page.Checkout.Confirmation" ||
b.device_type == "Mobile" && b.utagPageName == "page.Checkout.Confirmation.Mobile,F,70"))){
```